app/uk/gov/hmrc/cgtpropertydisposals/models/des/AddressDetails.scala

```diff
@@ -23,15 +23,15 @@ import cats.syntax.eq._
 import play.api.libs.json.{Json, OFormat}
 import uk.gov.hmrc.cgtpropertydisposals.models.address.Address.{NonUkAddress, UkAddress}
 import uk.gov.hmrc.cgtpropertydisposals.models.address.Country.CountryCode
-import uk.gov.hmrc.cgtpropertydisposals.models.address.{Address, Country}
+import uk.gov.hmrc.cgtpropertydisposals.models.address.{Address, Country, Postcode}
 import uk.gov.hmrc.cgtpropertydisposals.service.onboarding.BusinessPartnerRecordServiceImpl.Validation
 
 final case class AddressDetails(
   addressLine1: String,
   addressLine2: Option[String],
   addressLine3: Option[String],
   addressLine4: Option[String],
-  postalCode: Option[String],
+  postalCode: Option[Postcode],
```

Comment:
we should be able to put this back to `String` now to be safe

Reply:
done